docs/readmes/lte/dev_notes.md

```diff
@@ -282,3 +282,59 @@ enbs {
 }
 num_enbs: 1
 ```
+
+## Combining XML Unit test reports
+
+At the moment of writing, running the unit test targets does not generate the aggregated XML report file by default. To get the report of all unit tests, you have two options.  
+
+### Option 1: Running unit test target
+
+If you prefer a simple method, you can simply do:
+
+```bash
+cd ${MAGMA_ROOT}/lte/gateway/
+make test_oai_runtime
+```
+
+And you will get the combined report at `${MAGMA_ROOT}/report/merged_report/report_all_tests.xml`
+
+### Option 2: Manual method
```

Comment:
What is the benefit of specifying a manual method? As in, why would a developer opt to do this manually?

Reply:
Hi @themarwhal, my intention for this manual method is just in case, in the future, a developer would like to make some changes to my script and how to use it, they can use this method as a guideline to make their changes.